src/components/dashboard/main-content/course-enrollments/CourseAssignmentAlert.jsx

```diff
@@ -6,11 +6,12 @@ import { Info } from '@edx/paragon/icons';
 import { getContactEmail } from '../../../../utils/common';
 
 const CourseAssignmentAlert = ({
+  showAlert,
   onClose,
   variant,
 }) => {
-  const heading = variant === 'cancelled' ? 'Course assignment canceled' : 'Deadline passed';
-  const text = (variant === 'cancelled'
+  const heading = variant === 'canceled' ? 'Course assignment canceled' : 'Deadline passed';
+  const text = (variant === 'canceled'
```

Comment:
[curious] was this cancelled dependent on the APIs value being `cancelled`?

Reply:
Nope, this is a frontend-only  / hardcoded string passed to `CourseAssignmentAlert` from `CourseEnrollments`.